include/dav1d/picture.rs

```diff
@@ -289,6 +288,12 @@ pub struct Dav1dPicAllocator {
     ///
     /// If frame threading is used, accesses to [`Self::cookie`] must be thread-safe.
     ///
+    /// ### Additional `rav1d` requirement:
+    ///
+    /// The allocated data must be initialized.
+    /// If newly (e.x. not reused) allocated data is zero initialized using OS APIs,
+    /// this should not be slower than an uninitialized allocation.
```

Comment:
Also I don't think we should assert anything about zeroing being not slower than un-initialized allocation. Fresh pages sure, but some allocations reuse existing pages and will require a `memset` in `calloc`. In practice this should not be an issue as we have a stable set of picture buffers while decoding, but that doesn't generalize to all allocations.

Reply:
You can always request fresh pages with `mmap` or something, though.  You don't have to use `calloc`; `calloc` is just the easiest to switch to.

Reply:
What I'm saying is that for allocations < 1 page (and probably larger ones depending on the allocator) you won't, you'll re-use allocations.

Reply:
Can I change it to "meaningfully slower".  That is, if `calloc` decides reusing a page is faster, then the performance is probably small enough anyways.  Clearing a 4096-byte page should be pretty fast.

Reply:
I just don't think we should be asserting performance characteristics here at all without being very sure, and in this case that confidence isn't warranted.

Reply:
I still think it's a useful thing to point out generally, though.  Can I change it to "can be", because that's at least true on the benchmarks we've run?

Reply:
Reworded in [`b626855..fdb11e2`](https://github.com/memorysafety/rav1d/compare/b6268553bf189ba730e6ff162df91da20099e01d..fdb11e2add201f31c71f7ed9ea84f0260bad1f1d).  Is this good?